vagrant/post-deploy.d/20-setup-basic-auth.sh

```diff
@@ -2,6 +2,7 @@
 set -u
 set -e
 echo "Setting up concourse basic auth as ${CONCOURSE_ATC_USER}: ${CONCOURSE_ATC_PASSWORD}"
+sed "s/--external-url.*/--external-url \'\' \\\/" -i /var/vcap/jobs/atc/bin/atc_ctl # Remove hardcoded auth redirect IP
```

Comment:
Thanks.

For the benefit of anyone else following along: they have explicitly stated in that issue that they don't intend to build the AMI any longer.